app/tray/Badge/index.js

```diff
@@ -11,21 +11,21 @@ class Bridge extends React.Component {
         <div className='badgeWrap'>
           <div className='badge cardShow' style={{ transform: 'translateY(0px)', height: '196px' }}>
             <div className='badgeInner'>
-              <div className='badgeMessage'>Your update is ready, relaunch Frame to switch</div>
+              <div className='badgeMessage'>Your update is ready, relaunch Frame to switch?</div>
               <div className='badgeInput'>
                 <div className='badgeInputButton'>
                   <div
                     className='badgeInputInner'
-                    onMouseDown={() => link.send('tray:action', 'updateBadge', '')}
+                    onMouseDown={() => link.send('tray:updateRestart')}
                   >
                     Ok
```

Comment:
can we change this to say "Restart Now" to be consistent w. the previous label? also lets change "relaunch Frame" to "restart Frame" above as its more consistent with accepted verbiage in these types of prompts.